src/classes/weather_data_transform.py

```diff
@@ -0,0 +1,223 @@
+"""
+This module contains the WeatherDataTransform, which contains the transformations
+performed to the weather data parquet files
+"""
+import datetime
+
+import pandas as pd
+
+class WeatherDataTransform:
+    """
+    The methods in this class are used to transformed the weather data before
+    it is passed to the feature store
+
+    Methods:
+        * transform_individual_locations
+        * generate_all_information_df
+        * calculate_scaled_temperature
+        * get_max_weather
+    """
+
+    @staticmethod
+    def transform_individual_locations(file_path: str) -> pd.DataFrame:
+        """
+        Transform data from individual weather stations to remove duplicates,
+        NaNs, add the timestamp as the index and map the weather type
```

Comment:
Forgot to add that it also changes the data types of the columns and that it first create the date as day+hour from information in the parquet file

Reply:
Addressed